### ISSUE TITLE
Datasets used to train flownets_EPE1.951.pth

### ISSUE THREAD

**Issue by binue97** [ClementPinard/FlowNetPytorch#123]
First of all, great thanks to your work. :+1: 
I am quite new to ML/DL field and your project helped me a lot.

While exploring the issues tab, I noticed that the pre-trained weight provided (flownets_EPE1.951.pth) was trained using your custom hyperparameters, and some other details.

I was curious about the datasets and scheduling pipeline you used for training the model:
- Was it trained only on the FlyingChairs dataset?
- Or did it involve a combination, such as FlyingChairs + FlyingThings3D or even KITTI?

Any additional details about the training schedule would also be highly appreciated!
(For example, FlowNet2.0 paper mentioned that scheduling simple dataset first and then rather complicated dataset next showed better performance)

Thank you again for your contribution!

**Comment by ClementPinard** [comment]
Hello,  thank you for your interest in this repo :)

FlownetS was only trained on flyingchairs, nothing else. The reported EPE is also on Flying chairs validation set.

However, you can easily finetune this model for other datasets, such as KITTI or flying things 3D. You are right about the "simple" dataset : Flying chairs is a good dataset to be robust for random changes and only focus on what makes a optical flow, but it's not realistic. As such, finetuning this model with real images, allows the model to be both robust thanks to FlyingChairs pre-training, and precise, thanks to the finetuning real dataset.

Hope it helped you

**Comment by binue97** [comment]
Thank you for the detailed explanation! 😊
